cabal.project

```diff
@@ -12,16 +12,16 @@ repository cardano-haskell-packages
 source-repository-package
     type: git
     location: https://github.com/Plutonomicon/plutarch-plutus
-    tag: 7efbb9fc9f358cfbfd89ff245d85632a17ff3fb3
+    tag: fcdd2209433d8b8979e820dc4fa9aad5f202216d
     subdir:
       .
-      plutarch-api
       plutarch-extra
+      plutarch-ledger-api
 
 source-repository-package
     type: git
     location: https://github.com/input-output-hk/xsy-liqwid-libs
-    tag: 01dcaf60ee5386f0975aed908d6f92ab0e9b34e0
+    tag: 2425031da690546dd64669975598a99d1279eb16
```

Comment:
Depending on which PR we merge first, this could potentially revert the change on #21. So, I would change it to `f46485d8796db37a5b3e7fe90a830355504f4edb` instead.